.github/CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@
 - When developing a new feature, include at least one test when applicable.
 - When submitting a PR, please follow [this template](PULL_REQUEST_TEMPLATE.md) (which will probably be already filled up once you create the PR).
 - When submitting a PR with changes to user interface (e.g.: new screen, ...), please add screenshots to the PR description.
-- When submitting a PR, please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR), copy the link and place it in your PR.
+- When submitting a PR, please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/docs/wiki/Prepare-testing-GitHub-Pages-for-your-PR), copy the link and place it in your PR.
```

Comment:
Adding `docs` in this link does not have the intended effect. Because `<repository>/wiki` takes you to the [wiki page](https://github.com/anitab-org/anitab-org.github.io/wiki) of the repository on GitHub, and not the [Wiki file on the repository's source code](https://github.com/anitab-org/anitab-org.github.io/blob/develop/Wiki.md).

So in this case, you can either:
- not change the link and leave it to point towards the wiki
- or you can link to the file using this as the "link" `/docs/Wiki.md`